docs/modules/ROOT/pages/ios_installation.adoc

```diff
@@ -8,6 +8,6 @@ Next, log in and navigate to menu:Settings[General (Personal)].
 At the bottom of that page, you will see a link to the ownCloud app on iTunes.
 
 .Links to ownCloud's mobile apps.
-image:owncloud-server-mobile-apps.png[Links to ownCloud's mobile apps for Desktop, and in the Google Play and iOS App Stores.]
+image:owncloud-server-mobile-apps.png[Links.]
```

Comment:
Please don't make the `alt` attribute useless. It's an important accessibility feature.